README.md

```diff
@@ -378,6 +380,33 @@ local to the target Windows system.
 
    `winrm set winrm/config/winrs '@{MaxMemoryPerShellMB="1024"}'`
 
+* If you receive a timeout when trying to connect to your instance for the first time, make sure your Firewall setting
+  is permissive enough.
+
+   `netsh advfirewall firewall set rule name="Windows Remote Management (HTTP-In)" profile=public protocol=tcp localport=5985 remoteip=localsubnet new remoteip=any`
+
+### AWS User Data
+
+If you are spinning up AWS instances to test against, you can use the following user data when spinning up your instances:
+
+```
+<powershell>
+$logfile="C:\\Program Files\\Amazon\\Ec2ConfigService\\Logs\\kitchen-ec2.log"
+# Allow script execution
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Force
+#PS Remoting and & winrm.cmd basic config
+Enable-PSRemoting -Force -SkipNetworkProfileCheck
+& winrm.cmd set winrm/config '@{MaxTimeoutms="1800000"}' >> $logfile
+& winrm.cmd set winrm/config/winrs '@{MaxMemoryPerShellMB="1024"}' >> $logfile
+& winrm.cmd set winrm/config/winrs '@{MaxShellsPerUser="50"}' >> $logfile
+#Server settings - support username/password login
+& winrm.cmd set winrm/config/service/auth '@{Basic="true"}' >> $logfile
```

Comment:
With the current capabilities in the WinRM gem, we don't need the Basic='true' and AllowUnencrypted settings anymore.

I know I pointed you at an example from kitchen-ec2, but we've got to fix that there too. :)  Sorry.